tools/tools.go

```diff
@@ -86,6 +90,42 @@ func Sed(oldValue, newValue, filePath string) error {
 	return err
 }
 
+func SshRun(userName, userPassword, server, cmd string) (string, error) {
+	// Define ssh connection
+	config := &ssh.ClientConfig{
+		User: userName,
+		Auth: []ssh.AuthMethod{
+			ssh.Password(userPassword),
+		},
+		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
+	}
+
+	client, err := ssh.Dial("tcp", server+":22", config)
```

Comment:
nit: port should be configurable IMO

Reply:
agree

Reply:
Ok, I will do it, and let 22 as default if nothing is provided.

Reply:
In fact just add the port number in `server` like this `192.168.122.10:22` is enough, so will do this.